### PR TITLE
Add simple number converter GUI example

### PR DESCRIPTION
**Pull request by insomniacUNDERSCORElemon**
Other than what's already said in the readme (and visible w/example image)... the project shows signals, reading+changing data, procedures etc with notes on avoiding problems that I ran into. (also as noted in readme, it's hidden by default but there's some basic direct movement code there too)

Likely also simpler/more straightforward to understand in some ways than something dynamic (more moving parts) like a game.

--------------------------

Also I thought about doing a web export (Godot has been able to do that w/GDnative for a while) and linking it (on itch or gotm?) but I couldn't figure out how to get the export to work even on my system (app itself working via nimhttpd, no code working. I tried copying files/altering export as well).

### CODE REVIEW

**Comment by insomniacUNDERSCORElemon** [comment]
I also forgot to say: I thought about doing a web export (Godot has been able to do that w/GDnative for a while) and linking it (on itch or gotm?) but I couldn't figure out how to get the export to work even on _my_ system (app itself working via `nimhttpd`, no code working).

**Comment by zetashift** [comment]
Oh my! This looks cool, I haven't been working with Godot in a minute, I'll find the time to get to this PR this week and hopefully merge it right away.

Thank you!

**Comment by zetashift** [comment]
Looking good, except I see you use 1 space as indentation instead of 2? can you format the code using nimpretty or https://github.com/treeform/morepretty maybe?

**Comment by insomniacUNDERSCORElemon** [comment]
Done.

File: decbinhex/src/main.nim [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+  proc submit_binary() {.gdExport.} =
+    self.getNode("grid/a/outlabel").Label.text = convert_from_binary(
+      self.getNode(
+      "grid/inbox").LineEdit.text)
+
+  proc submit_hex() {.gdExport.} =
+    self.getNode("grid/a/outlabel").Label.text = convert_from_hexadecimal(
+      self.getNode("grid/inbox").LineEdit.text)
+    #self.getNode("grid/inbox").queue_free()
```
Comment (zetashift) [inline decbinhex/src/main.nim]: Why is this commented out?

Comment (insomniacUNDERSCORElemon) [reply decbinhex/src/main.nim]: Just something I tried, not relevant to the project.

Comment (insomniacUNDERSCORElemon) [reply decbinhex/src/main.nim]: Did you see that I removed the comment?

**Comment by zetashift** [comment]
Thank you very much! :D

**Pull request merged**